build_infrastructure/meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+galaxy_info:

```

Comment:
Seems to be info for `darkwizard242.terraform` role, please change to be our own